napari/_vispy/overlays/scale_bar.py

```diff
@@ -190,7 +190,9 @@ def _on_position_change(self, event=None):
         # prevent the text from being cut off by shifting down
         if 'top' in self.overlay.position:
             # convert font_size to logical pixels as vispy does
-            # 96 dpi is used as the reference dpi
+            # in vispy/visuals/text/text.py
+            # 72 is the vispy reference dpi
+            # 96 dpi is used as the napari reference dpi
```

Comment:
so yeah, 72 is the "default" physical dpi, 96 is the "default" logical dpi (see mi comment above and https://en.wikipedia.org/wiki/Dots_per_inch#Computer_monitor_DPI_standards)